.github/workflows/test.yml

```diff
@@ -65,23 +59,15 @@ jobs:
     
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: '3.x'
       - uses: actions/setup-node@v1
-        with:
-          node-version: '10'
-
-      - name: Install Postgres Requirements
-        run: sudo apt-get install postgresql postgresql-contrib libpq-dev
     
       - name: Install and resolve dependencies
         run: |
-          yarn
+          yarn --frozen-lockfile
           node testing.js
 
       - name: Run Application Server for DuckDB
-        run: node server.js db &
```

Comment:
Won't this be needed for the server to run in the background ?

Reply:
I missed that, thanks. Updated.

Reply:
What do you think now?